demos/vc_issuer/tests/issue_credential.rs

```diff
@@ -371,23 +371,6 @@ fn should_return_derivation_origin_with_custom_init() {
     assert_eq!(response.origin, custom_init.derivation_origin);
 }
 
-#[test]
```

Comment:
Please annotate with `#[ignore]` rather than deleting.

Reply:
Can you explain why that's better? If the test needs to be resurrected it can always be found in the git history

Reply:
It is more convenient to re-enable and is nagging in the test output to do something about it.

Reply:
>  is nagging in the test output to do something about it.

That's unfortunate! I think the cleanest way would be to track this somewhere, otherwise (by ignoring the test) it might never get done though it will always be a slight annoyance when running the tests. And in case we never re-add the test, having it ignored is just wasted real estate in the file! Nothing as permanent as temporary solutions.